packages/view-types/feature-list/src/selectable-table/FeatureListOptions.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { OptionsContainer, OptionSelect, usePlotOptionsStyles } from '@vitessce/vit-s';
+import { TableCell, TableRow, Checkbox } from '@material-ui/core';
+import { FEATURELIST_SORT_OPTIONS } from './constants.js';
+
+
+export default function FeatureListOptions(props) {
+  const {
+    children,
+    featureListSort,
+    setFeatureListSort,
+    showFeatureTable,
+    setShowFeatureTable,
+  } = props;
+
+  function handleFeatureListSortChange(event) {
+    setFeatureListSort(event.target.value);
+  }
+
+  function handleShowTableChange(event) {
+    setShowFeatureTable(event.target.checked);
+  }
+
+  const classes = usePlotOptionsStyles();
+
+  return (
+    <OptionsContainer>
+      {children}
+      <TableRow>
+        <TableCell className={classes.labelCell} htmlFor="feature-list-sort-option-select">
+          Sort features by
+        </TableCell>
```

Comment:
Can we add a third `<select/>`, called "Sort Key" with options `Gene ID` and `Alternate ID`? Otherwise it might be confusing which column is used for sorting.

(also with the option to pass down an initial value via the `FeatureListSubscriber` props like you have done with `showTable` and `showFeatureTable`)

Reply:
Yes, good point. I have added a new select, which is disabled if the ordering is not `alphabetical`